sml_builder/static/js/cookies_function.js

```diff
@@ -0,0 +1,184 @@
+const DEFAULT_COOKIE_CONSENT = {
+    essential: true,
+    settings: false,
+    usage: true,
```

Comment:
Sorry - having re-read the cookie policy in the design system I think it is correct that usage is false by default. The only cookie set by default should be essential. My misunderstanding of the context of the cookies policy.

Reply:
Reverted :)